atuin/src/command/client/history.rs

```diff
@@ -367,13 +386,56 @@ impl Cmd {
         Ok(())
     }
 
-    pub async fn run(self, settings: &Settings, db: &impl Database) -> Result<()> {
+    async fn init_store(
+        context: atuin_client::database::Context,
+        db: &impl Database,
+        store: HistoryStore,
+    ) -> Result<()> {
+        println!("Importing all history.db data into records.db");
+
+        let history = db
+            .list(
+                atuin_client::settings::FilterMode::Global,
+                &context,
+                None,
+                false,
+                true,
+            )
+            .await?;
+
+        for i in history {
+            println!("loaded {}", i.id);
+
+            if i.deleted_at.is_some() {
+                store.push(i.clone()).await?;
+                store.delete(i.id).await?;
+            } else {
+                store.push(i).await?;
+            }
+        }
+
+        Ok(())
+    }
+
+    pub async fn run(
+        self,
+        settings: &Settings,
+        db: &impl Database,
+        store: SqliteStore,
+    ) -> Result<()> {
         let context = current_context();
 
+        let encryption_key: [u8; 32] = encryption::load_key(settings)
+            .context("could not load encryption key")?
+            .into();
```

Comment:
will break for non-sync users, right?

Reply:
Nope - `load_key` will generate a key if one doesn't exist. Regardless of sync, the store should save all data as encrypted